client/src/components/MusicPlayer.jsx

```diff
@@ -1,51 +1,68 @@
 import React from 'react';
 // import CSSModules from 'react-css-modules';
-import Equalizer from './Equalizer.jsx';
+import Waves from './Waves.jsx';
 import styles from '../styles/MusicPlayer.css';
 import axios from 'axios';
 import TimeAgo from 'react-timeago';
 
-
-const imagePaths = {
-  pauseButton: 'https://s3-us-west-1.amazonaws.com/democrituscloud/pause.png',
-  playButton: 'https://s3-us-west-1.amazonaws.com/democrituscloud/play.png'
-}
+var audio;
```

Comment:
just curious, what were the tradeoffs for having `audio` as a global var instead as part of state?